aries/aries_vcx_core/src/wallet/entry_tag.rs

```diff
@@ -77,3 +110,42 @@ impl From<EntryTags> for Vec<EntryTag> {
         value.inner
     }
 }
+
+pub fn pair_into_entry_tag(pair: (String, String)) -> EntryTag {
+    if pair.0.starts_with('~') {
+        EntryTag::Plaintext(pair.0.trim_start_matches('~').into(), pair.1)
+    } else {
+        EntryTag::Encrypted(pair.0, pair.1)
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use crate::wallet::entry_tag::{EntryTag, EntryTags};
+
+    #[test]
+    fn should_serialize_entry_tags() {
+        let tags = EntryTags::new(vec![
+            EntryTag::Plaintext("a".into(), "b".into()),
+            EntryTag::Encrypted("c".into(), "d".into()),
+        ]);
+
+        let res = serde_json::to_string(&tags).unwrap();
+
+        assert_eq!("{\"~a\":\"b\",\"c\":\"d\"}", res);
+    }
+
+    #[test]
+    fn shoud_deserialize_entry_tags() {
```

Comment:
typo. Also, it's better to make it easy to group the test names based on prefixes, e.g. `test_entry_tags_serialize` and `test_entry_tags_deserialize`, because then you can run `cargo test -- test_entry_tags` to run both tests.

Reply:
Changed.